stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/FileAttachmentsView.kt

```diff
@@ -273,13 +272,12 @@ private class FileAttachmentViewHolder(
             fileSizeView: TextView,
             progressBar: ProgressBar,
             attachment: Attachment,
-            progress: Int,
+            uploadedBytes: Long,
             isComplete: Boolean,
-            progressCorrection: Float,
             targetValue: String,
         ) {
             if (!isComplete) {
-                val nominalProgress = MediaStringUtil.convertFileSizeByteCount((progress * progressCorrection).toLong())
```

Comment:
This was simply an incorrect calculation, getting us stuck permanently at 0 for this value on the UI